contracts/Vault.sol

```diff
@@ -299,9 +288,7 @@ contract Vault is Auth {
         address asset,
         address positionOwner,
         uint mainAssetToLiquidator,
-        uint colToLiquidator,
         uint mainAssetToPositionOwner,
-        uint colToPositionOwner,
         uint repayment,
```

Comment:
During liquidation, accrued stability fee is included into `repayment` and gets burnt, not transferred to the foundation as usual. Is it the right behavior?